### ISSUE TITLE
Creating lock fails on Python 3.9.7

### ISSUE THREAD

**Issue by radcliffkey** [elarivie/pyReaderWriterLock#11]
`rwlock.RWLockFair()` raises

```
...
File "/usr/local/lib/python3.9/site-packages/readerwriterlock/rwlock.py", line 321, in __init__
    super().__init__()
File "/usr/local/lib/python3.9/typing.py", line 1083, in _no_init
    raise TypeError('Protocols cannot be instantiated')
```

This is probably related to Python change https://bugs.python.org/issue44806 in Python 3.9.7 https://docs.python.org/release/3.9.7/whatsnew/changelog.html

**Comment by elarivie** [comment]
Hello @radcliffkey,

Thanks for the bug report

I did reproduce the issue and it really seems to be related to the changes within Python version 3.9.7 as you identified.

I am looking into it

**Comment by elarivie** [comment]
Hello @radcliffkey 

This is now fixed by 516eb0f1e6ea5e0fdae5362c9c281249802261f3 and available in [readerwriterlock version 1.0.9](https://pypi.org/project/readerwriterlock/)

**Comment by radcliffkey** [comment]
Great. Thank you for a quick reaction!
